### ISSUE TITLE
NPM install 1.4.0 not working

### ISSUE THREAD

**Issue by johanlajili** [pixijs/pixi-spine#151]
registry error parsing json
```
npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/Cellar/node/6.5.0/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v6.5.0
npm ERR! npm  v3.10.3

npm ERR! Unexpected token < in JSON at position 0
npm ERR! <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
npm ERR! <html><head>
npm ERR! <title>404 Not Found</title>
npm ERR! </head><body>
npm ERR! <h1>Not Found</h1>
npm ERR! <p>The requested URL /content/groups/npm-group/@types/pixi.js was not found on this server.</p>
npm ERR! <hr>
npm ERR! <address>Apache/2.2.15 (CentOS) Server at nexus.gamesys.co.uk Port 443</address>
npm ERR! </body></html>
npm ERR!
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/johan.lajili/Documents/workspace/warp/slot-jins-bouncing-wilds/npm-debug.log
```

**Comment by cursedcoder** [comment]
```<address>Apache/2.2.15 (CentOS) Server at nexus.gamesys.co.uk Port 443</address>```

it seems you have some problems with firewall or something

**Comment by johanlajili** [comment]
Hey, coming back on that month later. This was caused by our NPM registry (nexus) not supporting the Scope Package feature 
````json
"depenencies": {
    "@types/pixi.js": "4.3.1"
}
````
( in the package.json).
Since we are not in a position to update that registry, and are planned to switch to a new system in the near future, we decided to fork the project and change the package.json like so, in our private github.
````json
"dependencies": {
    "pixi.js": "4.3.1"
  }
````
Not asking you to do so but leaving it here for other people having the issue in the future.
